tests/api_connexion/endpoints/test_dag_endpoint.py

```diff
@@ -387,6 +396,7 @@ def test_should_raise_404_when_dag_is_not_found(self):
 class TestGetDags(TestDagEndpoint):
     def test_should_respond_200(self):
         self._create_dag_models(2)
+        self._create_deactivated_dag()
```

Comment:
How about you add a query that asserts that the number of dags in DagModel is  3

Reply:
Number of DAGs in DagModel should exclude inactive DAGs, as far as I understand. In my opinion are inactive DAGs `.trash` and play no significant role.

Reply:
I know but how are we sure that an inactive dag exists here? Do we have a test for the new method you created? I think that querying DagModel and asserting that it has 3 dags(both active and inactive) at this point will help to know that the method you created works fine. If there's an alteration in the method in the future to work another way, it'll be noticed. What do you think?

Reply:
That makes sense, should I query DagModel at the same place where we are testing the API Endpoint?